packages/babel-plugin-minify-simplify/README.md

```diff
@@ -2,9 +2,10 @@
 
 This plugin will transform code in mainly two ways:
 
-1. Reduce as much statements as possible into expressions
+1\. Reduce as much statements as possible into expressions
```

Comment:
It's a temporary workaround to get the numbering correct, currently:

![image](https://cloud.githubusercontent.com/assets/56288/20889707/a5db45cc-bac9-11e6-8902-0740656b0104.png)